game.py

```diff
@@ -2043,7 +2082,7 @@ def _generate_scorechart(self):
         total_games = 1
         all_numbers = []
 
-        for key in self.all_scores.keys():
+        for key in list(self.all_scores.keys()):
```

Comment:
Is this `list()` needed, can we just use:
```python
for key in self.all_scores.keys():
    ...
```

Reply:
Same as I pointed out above.